Description: Correct pod NAME issue in LedgerSMB/RESTXML/Handler.pm
 Is missing the pod NAME entry altogether.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472616&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/RESTXML/Handler.pm
+++ b/LedgerSMB/RESTXML/Handler.pm
@@ -1,3 +1,11 @@
+=pod
+
+=head1 NAME
+
+LedgerSMB::RESTXML::Handler - RESTful LSMB API
+
+=cut
+
 package LedgerSMB::RESTXML::Handler;
 use strict;
 use warnings;
